tests/data/transformer/14-mixed-operators/authorization-model.fga

```diff
@@ -0,0 +1,13 @@
+model
+  schema 1.1
+
+type user
+
+type document
+  relations
+    define allowed: [user]
```

Comment:
@d-jeffery these will have to be reordered alphabetically to pass the transformation back into DSL, but leave this part till last.

First, ensure transformation into JSON is working as expected for JS